packages/react-components/src/components/chart/legend/legend.tsx

```diff
@@ -5,12 +5,19 @@ import useChartsLegend from '../hooks/useChartsLegend';
 import { SeriesOption } from 'echarts';
 
 import './legend.css';
+import { useCollection } from '@cloudscape-design/collection-hooks';
 
 const Legend = (legendOptions: { series: SeriesOption[]; graphic: InternalGraphicComponentGroupOption[] }) => {
-  const { items, columnDefinitions } = useChartsLegend(legendOptions);
+  const { items: allItems, columnDefinitions } = useChartsLegend(legendOptions);
+
+  const { items, collectionProps } = useCollection(allItems, {
+    sorting: {},
```

Comment:
Does this apply any default sort based on the behaviour of `useCollection`? Any behaviour change that we can test?

Reply:
useCollection is from cloudscape so did not add any specific test for sorting, given its handled outside our component

Reply:
no default sort is applied, we could, but letting it to render the initial data as is